domain/src/main/kotlin/no/nav/su/se/bakover/domain/oppdrag/Utbetaling.kt

```diff
@@ -4,16 +4,19 @@ import no.nav.su.se.bakover.common.Tidspunkt
 import no.nav.su.se.bakover.common.UUID30
 import no.nav.su.se.bakover.domain.Fnr
 import no.nav.su.se.bakover.domain.NavIdentBruker
+import no.nav.su.se.bakover.domain.Saksnummer
 import no.nav.su.se.bakover.domain.oppdrag.avstemming.Avstemmingsnøkkel
 import no.nav.su.se.bakover.domain.oppdrag.simulering.Simulering
+import java.util.UUID
 
 sealed class Utbetaling {
     abstract val id: UUID30
     abstract val opprettet: Tidspunkt
+    abstract val sakId: UUID
```

Comment:
Hvorfor trenger vi begge disse?

Reply:
Hvorman både trenger `saksnummer` og `sakId` på Utbetaling-typen?

Vi trenger `saksnummer` siden det er det vi sender som `fagsystemId` og `refFagsystemId` til oppdrag. Vi kunne selvfølgelig flyttet denne ut av Utbetaling og latt den flyte ved siden av til Utbetalingsstrategi, f.eks. ved at vi sendte med Sak som en parameter og enten så wrappet den eller sendte Sak helt inn til UtbetalingMapper. 

Vi trenger `sakId` for å persistere, siden `oppdragId` er slettet fra utbetaling-tabellen og erstattet med sakId som foreign key. `sakId` er vår interne id. Dog, vi hadde bare trengt den ved insert, men har ikke skilt på command and query når det kommer til Utbetaling.

Reply:
Ingen problemer med å forstå hvorfor vi har begge altså, men jeg hadde vel kanskje sett for meg at det holdt med saksnummer og at vi alltid kunne finne fram til `sakId` ved hjelp av den ved behov (f.eks ved persistering). I mitt hode er saksnummer dermed fellesnevneren vi trenger for å oppfylle alle behovene (våre og oppdrag).